app/View/NotFound.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+class NotFoundView extends React.Component {
+    render() {
+        return (
+            <div>
+                <div className="row">
+                    <div className="col-lg-12">
+                        <div className="page-header">
+                            <h1>Not Found</h1>
+                        </div>
+                    </div>
+                </div>
+
+                <div className="row dashboard-content">
+                    <div className="col-lg-12">
+                        The page you are looking for cannot be found. Take a break before trying again.
+                        <br/>
+                        <br/>
+                        <br/>
+                        <br/>
+                    </div>
+
+                    <pre className="ascii col-lg-offset-4 col-lg-4 col-md-offset-3 col-md-6 col-sm-offset-2 col-sm-8 col-cs-offset-1 col-xs-10">
+<br/>
```

Comment:
I had issue without it, maybe because of the compilation step (made by webpack).